src/server/internalServer.cpp

```diff
@@ -741,8 +741,10 @@ std::string get_book_name(const RequestContext& request)
   }
 }
 
-std::string searchSuggestionHTML(const std::string& searchURL, const std::string& pattern)
+std::string searchSuggestionHTML(std::string& searchURL, const std::string& pattern)
 {
+  searchURL = replaceRegex(searchURL, "&lt;", "<");
+  searchURL = replaceRegex(searchURL, "&gt;", ">");
```

Comment:
We should replace `<` and `>` with the url encoding equivalent, not the html encoding.
This is a url and we want it to work for articles with `<`/`>` in it (at least be able to search for them)